kong/runloop/handler.lua

```diff
@@ -91,7 +91,7 @@ do
     local time = ngx.time()
 
     if force or time - last >= LUA_MEM_SAMPLE_RATE then
-      local count = collectgarbage("count")
+      local count = collectgarbage("count") * 1024
```

Comment:
Actually, could we fix this on the PDK side instead of within the handler? This way, we ensure that GC usage values can be reported in kb from anywhere (without additional processing), and the PDK is made aware that all values reported as such are in kb, and would be the only place where we make the kb -> b conversion.

Reply:
Sure, I will rework the fix and test.

Reply:
@thibaultcha could you re-review? Would love this to make it into 2.1.1. Thanks!